JournalCoreData/Model Controllers/EntryController.swift

```diff
@@ -108,7 +111,7 @@ class EntryController {
                 completion(NSError())
                 return
             }
-
+            
             let moc = CoreDataStack.shared.mainContext
```

Comment:
Here is one you missed. This is being run on a background thread... Therefore you need to create a background context and use `perform` or `performAndWait` (which is already being done, so just need to use a proper context)